packages/squiggle-lang/__tests__/TS/KDE.ts

```diff
@@ -0,0 +1,59 @@
+import * as fc from "fast-check";
+import { samplesToContinuousPdf } from "../../src/rescript/Distributions/SampleSetDist/KdeLibrary.js";
+import range from "lodash/range";
+
+const evalKde = (samples: number[], width: number, weight: number, x: number) =>
```

Comment:
I feel like this function would take me some time to understand what it's doing and how it works. I'd imagine some comments/cleanup would help here.

Reply:
Hm, how long? It's a direct translation of the [standard KDE definition](https://en.wikipedia.org/wiki/Kernel_density_estimation#Definition) with a triangular K, x goes to max(0, |1-x|). If I have to explain background knowledge at that level it's going to take up a lot of development time.

Reply:
Maybe link to that page, like, 
"Follows the algorithm here".

I'm used to thinking of "KDE" as returning a full pointSet, it wasn't super obvious this meant if for one point x. (I could see the argument, but there are 4 args)